pkg/commands/service/service_test.go

```diff
@@ -71,6 +71,87 @@ func TestServiceCreate(t *testing.T) {
 	}
 }
 
+type mockServicesPaginator struct {
```

Comment:
Here is an example set of tests to validate the pagination logic. 

It's for the `service list` command but `acl-entry list` and `dictionary-item list` have the exact same tests.